.spelling

```diff
@@ -370,6 +370,8 @@ json
 JSON-formatted
 jwcrypto
 JWT
+JWTRule
```

Comment:
Instead of adding all these words to .spelling, you should just put backtick characters around them in the doc: e.g. `JWTRule`.

Reply:
Great just updated this in the new commit, thanks!